app/main.py

```diff
@@ -1 +1,49 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        super().learn_skill("JavaScript")
+        super().learn_skill("HTML")
+        super().learn_skill("CSS")
+
```

Comment:
```suggestion
        self.skills.extend(["JavaScript", "HTML", "CSS"])

```

Reply:
Instead of calling super().learn_skill() multiple times, use `self.skills.extend()` to add multiple skills at once.